### ISSUE TITLE
Lights RAD export feature

### ISSUE THREAD

**Issue by Qwertyus3D** [wootguy/bspguy#77]
It could be useful to have an export option of the existing information about textures lighting into RAD file, to make possible a recreation of original lighting styles for decompiled maps.

**Comment by UnrealKaraulov** [comment]
I think you can't do it, because after recompile, you got another map structure, I think this is impossible.

**Comment by wootguy** [comment]
yeah, the texture light values are gone after the map is compiled. You could maybe get approximate values by checking the lightmap colors on the texture light faces though. Probably the values would be too bright from other lights shining onto the texture light faces.

**Comment by wootguy** [comment]
Texlights can be estimated in the new "RAD Preparation" tool. Texlights are written to an info_texlights entity in the map instead of exported to a file. Colors may be slightly off, and brightness will be very wrong, but this can give you a good starting point.

Some texlights won't be detected if other lights are shining onto them to create inconsistent lightmaps (+0~LAB_CRT8 in c1a0). Normal textures are falsely detected as texlights if they have perfectly consistent lightmaps (out_box2 in c2a4b).

You're asked if you want to compare with a recompiled map to remove some false positives. The recompiled map should not use a lights.rad file. Texlights identified in both maps cannot possibly be texlights if only one map had texlights enabled. In most cases it's probably faster to inspect each texture and remove the false positives manually.
